curriculum/challenges/english/25-front-end-development/quiz-css-typography/66ed9010f45ce3ece4053eb8.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --distractors--
 
-Placeholder distractor 1
+The space between lines of text
 
 ---
 
-Placeholder distractor 2
+The overall spacing of a text block
 
 ---
 
-Placeholder distractor 3
+The size of the font
 
 #### --answer--
 
-Placeholder answer
+The space between individual letters
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What’s the most common way to add Google Fonts to a website?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use the `<font>` tag in the HTML file
 
 ---
 
-Placeholder distractor 2
+Use the `@import` rule in JavaScript
 
 ---
 
-Placeholder distractor 3
+Embed the font within an image
 
 #### --answer--
 
-Placeholder answer
+Use the `<link>` tag in the HTML file
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How do you import the Roboto font into your CSS using the `@import` rule?
 
 #### --distractors--
 
-Placeholder distractor 1
+`@font-face { font-family: 'Roboto'; url: 'https://fonts.googleapis.com/css2?family=Roboto&display=swap'; }`
 
 ---
 
-Placeholder distractor 2
+`link: 'https://fonts.googleapis.com/css2?family=Roboto';`
 
 ---
 
-Placeholder distractor 3
+`@import google-font('Roboto');`
 
 #### --answer--
 
-Placeholder answer
+`@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which one of the following is a web-safe font?
 
 #### --distractors--
 
-Placeholder distractor 1
+Comic Sans MS
 
 ---
 
-Placeholder distractor 2
+Papyrus
 
 ---
 
-Placeholder distractor 3
+Impact
 
 #### --answer--
 
-Placeholder answer
+Times New Roman
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `@font-face` rule do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+It defines the font size
 
 ---
 
-Placeholder distractor 2
+It creates a fallback font for web pages
 
 ---
 
-Placeholder distractor 3
+It adjusts the line height for text
 
 #### --answer--
 
-Placeholder answer
+It allows you to use custom fonts by importing them
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which service offers a large selection of free fonts for use on websites?
 
 #### --distractors--
 
-Placeholder distractor 1
+Typekit
 
 ---
 
-Placeholder distractor 2
+Adobe Fonts
 
 ---
 
-Placeholder distractor 3
+Font Squirrel
 
 #### --answer--
 
-Placeholder answer
+Google Fonts
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is it important to include fallback fonts in your CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+It reduces page load times
 
 ---
 
-Placeholder distractor 2
+It prevents browser compatibility issues
 
 ---
 
-Placeholder distractor 3
+It gives the page more visual variety
 
 #### --answer--
 
-Placeholder answer
+It ensures a consistent experience in case the custom font doesn’t load
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How many values does the `text-shadow` property usually have?
```

Comment:
I don't quite like this question. Traditionally speaking, yes, developers probably want more than just the two required offset values but it feels unverifiable.